apps/docs/docs/how-to-guides/platform-integration/vtex/hosting-a-faststore-vtex-website.md

```diff
@@ -1,74 +1,72 @@
+- [Set up your **VTEX** account.](/how-to-guides/platform-integration/vtex/setting-up-an-account)
+- Develop and deploy your FastStore project.
+- Buy the desired domain.
+- [Redirect the **root domain** to the `www` subdomain.](https://help.vtex.com/en/tutorial/configuring-access-without-www--tutorials_4278#root-domain-redirect-without-www)
 
 ---
 
 ## Step by step
 
-### Step 1 - Getting the IP address of your website
-
-Use the `nslookup` command as in the following to check which **IP address** corresponds to your store website:
-
-1. Open the terminal.
-2. Check which IP addresses are capable of hosting your website by running the following command:
-
-- Remember to replace `{URL}` with your automatic deployment URL (e.g., `base.vtex.app`, `base.netlify.app`, `base.vercel.app`).
-
-```
-nslookup {URL}
-```
-
-4. Copy and save one of the IP addresses presented below the `Non-authoritative answer` message for further usage.
-
-![Getting the IP address of your website via the terminal](https://vtexhelp.vtexassets.com/assets/docs/src/nslookup___388da0c4d31ac04a2979dd6435107f93.png)
-
-### Step 2 - Configuring your website domain
-
-Now, it's time to make your website publicly available on the internet.
-
 :::caution
-If you still need to configure other VTEX solutions, please **use a fictitious domain name** when taking the following steps. Then, once you set up all the desired integrations, repeat this guide using your final domain to make your website publicly available to end-users.
+If you still need to configure other VTEX solutions, please **use a fictitious domain name** while taking the following steps. Then, once you set up all the desired integrations, repeat this guide using your final domain to make your website publicly available to end-users.
 :::
 
+### Step 1 - Creating the DNS records
+
 1. Access your domain provider website.
-2. Configure a **domain name** (for example, `vtexfaststore.com`) that points to the **IP address** you saved in the previous step. _Check the documentation of your domain provider for more information._
-3. Now, configure a **subdomain** named `secure` that points to `secure.{rootDomain}.cdn.vtex.com`, where `{rootDomain}` is the **complete** address of your website. For example, `secure.vtexfaststore.com.cdn.vtex.com`. This subdomain will be used by the **Checkout**, **Order Placed**, **Login**, and **My Account** pages.
+2. Create a **CNAME** DNS record for your domain that points to `{rootDomain}.cdn.vtex.com`, where `{rootDomain}` is the **complete** address of your website (e.g., `mystore.com`). _Check the documentation of your domain provider for more information._
+3. Now, create a **subdomain** record named `secure` that points to `secure.{rootDomain}.cdn.vtex.com`. For example, `secure.mystore.com.cdn.vtex.com`. This subdomain will be used by the **Checkout**, **Order Placed**, **Login**, and **My Account** pages.
```

Comment:
I think it would be more concise to specify the subdomain to be created here, as the `secure.mystore.com.cdn.vtex.com` is the subdomain used by VTEX, WDYT?

```suggestion
3. Now, create a **subdomain** record named `secure` (for example, `secure.mystore.com`) that points to `secure.{rootDomain}.cdn.vtex.com`. This subdomain will be used by the **Checkout**, **Order Placed**, **Login**, and **My Account** pages.
```